buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -432,7 +432,7 @@ class BeamModulePlugin implements Plugin<Project> {
         commons_compress                            : "org.apache.commons:commons-compress:1.19",
         commons_csv                                 : "org.apache.commons:commons-csv:1.4",
         commons_io                                  : "commons-io:commons-io:2.6",
-        commons_lang3                               : "org.apache.commons:commons-lang3:3.6",
+        commons_lang3                               : "org.apache.commons:commons-lang3:3.9",
```

Comment:
You could remove this from this list and have the full maven coordinates appear in sdks/java/core/build.gradle since it is no longer used elsewhere.

Reply:
Done